docs/cookbook/deployment/sylius-cloud-management-basics.rst

```diff
@@ -0,0 +1,232 @@
+Sylius Cloud by Platform.sh management basics
+=============================================
+
+Creating a new project and environment
```

Comment:
we have creating a new project in 2 places 🤔 I would be after sticking it to the preparing the project, environments (that are actually branches of the existing gh repo - called project) could be a different section